pym/bob/input.py

```diff
@@ -2463,6 +2489,9 @@ def __parse(self):
         self.__createSchemas()
 
         # user config(s)
+        self.__parseUserConfig("/etc/bobdefault.yaml")
```

Comment:
I think we got a problem here with possible includes of other files. Currently they are included relative to the current working directory. That's broken for such global files.

The most sane solution would be to do it again like git where includes are done relative to the current file. Unfortunately this will break backwards compatibility. Another solution could be to invent a special syntax like `./` for relative includes. But this is easy to get wrong and comes unexpected to the uninformed user. Thoughts?